Automattic/jetpack
# Boost: fix no-boost tooltip overlay issue

#### Changes proposed in this Pull Request:
Fix tooltip on no-boost score to appear above label on mobile.

#### Jetpack product discussion
p1633441358008100-slack-C0232B8CUHG

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Open boost option page on mobile and check if the tooltips are still under the score label.

##### Before:
![Before](https://cldup.com/WHKI_6_Kjg.png)

##### After:
![After](https://user-images.githubusercontent.com/3737780/136339549-974e486b-811d-4cfd-b960-a6c431ad92b5.png)
